color_map: #bgr
    color0: [0, 0, 0]
    color1: [0, 0, 255]
    color10: [245, 150, 100]
    color11: [245, 230, 100]
    color13: [250, 80, 100]
    color15: [150, 60, 30]
    color16: [255, 0, 0]
    color18: [180, 30, 80]
    color20: [255, 255, 0]
    color30: [30, 30, 255]
    color31: [200, 40, 255]
    color32: [90, 30, 150]
    color40: [255, 0, 255]
    color44: [255, 150, 255]
    color48: [75, 0, 75]
    color49: [75, 0, 175]
    color50: [0, 200, 255]
    color51: [50, 120, 255]
    color52: [0, 150, 255]
    color60: [170, 255, 150]
    color70: [0, 175, 0]
    color71: [0, 60, 135]
    color72: [80, 240, 150]
    color80: [150, 240, 255]
    color81: [0, 0, 255]
    color99: [255, 255, 50]
    color252: [245, 150, 100]
    color256: [255, 0, 255]
    color253: [200, 40, 255]
    color254: [30, 30, 255]
    color255: [90, 30, 150]
    color257: [250, 80, 100]
    color258: [180, 30, 80]
    color259: [255, 255, 0]

label_map: 
    label0 : "unlabeled"
    label1 : "outlier"
    label10: "car"
    label11: "bicycle"
    label13: "bus"
    label15: "motorcycle"
    label16: "on-rails"
    label18: "truck"
    label20: "other-vehicle"
    label30: "person"
    label31: "bicyclist"
    label32: "motorcyclist"
    label40: "road"
    label44: "parking"
    label48: "sidewalk"
    label49: "other-ground"
    label50: "building"
    label51: "fence"
    label52: "other-structure"
    label60: "lane-marking"
    label70: "vegetation"
    label71: "trunk"
    label72: "terrain"
    label80: "pole"
    label81: "traffic-sign"
    label99: "other-object"
    label252: "moving-car"
    label253: "moving-bicyclist"
    label254: "moving-person"
    label255: "moving-motorcyclist"
    label256: "moving-on-rails"
    label257: "moving-bus"
    label258: "moving-truck"
    label259: "moving-other-vehicle"